tests/test_transforms/test_math/test_limit_transform.py

```diff
@@ -0,0 +1,146 @@
+    )
+    df["non_target_transformed_3"] = np.array(
+        [1.6094379124541003, 1.7917594692447216, 1.945910149069599, 2.079441541692336]
+    )
+    df["non_target_transformed_4"] = np.array([-0.6931471805349453, 0.0, 0.6931471805349453, 1.6094379123541003])
+    df = TSDataset.to_dataset(df)
+    tsds = TSDataset(df, freq="D")
+    return tsds
+
+
+@pytest.mark.parametrize(
+    "lower_bound,upper_bound,n_test", [(None, None, "1"), (None, 5, "2"), (-1, None, "3"), (0, 5, "4")]
+)
+def test_fit_transform_target(ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str):
+    """Check forward scaled logit transform works correctly on target column"""
+    preprocess = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["target"], result["segment_1"]["target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize(
+    "lower_bound,upper_bound,n_test", [(None, None, "1"), (None, 9, "2"), (0, None, "3"), (3, 9, "4")]
+)
+def test_fit_transform_non_target(
+    ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str
+):
+    """Check forward scaled logit transform works correctly on nontarget column"""
+    preprocess = LimitTransform(in_column="non_target", lower_bound=lower_bound, upper_bound=upper_bound)
+    result = preprocess.fit_transform(ts=ts_check_limit_transform).to_pandas()
+    np.testing.assert_array_almost_equal(
+        result["segment_1"]["non_target"], result["segment_1"]["non_target_transformed_" + n_test]
+    )
+
+
+@pytest.mark.parametrize("lower_bound,upper_bound", [(None, None), (None, 9), (0, None), (0, 9)])
+def test_inverse_transform(ts_check_limit_transform: TSDataset, lower_bound: float, upper_bound: float):
+    """Check that inverse_transform rolls back transform result"""
+    df_copy = ts_check_limit_transform.to_pandas()
+    preprocess = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    preprocess.fit_transform(ts=ts_check_limit_transform)
+    preprocess.inverse_transform(ts=ts_check_limit_transform)
+    result = ts_check_limit_transform.to_pandas()
+    np.testing.assert_array_almost_equal(result["segment_1"]["target"], df_copy["segment_1"]["target"])
+
+
+@pytest.mark.parametrize(
+    "lower_bound,upper_bound,n_test", [(-10, 200, "1"), (-10, None, "2"), (-30, 50, "3"), (None, 50, "4"), (-5, 5, "5")]
+)
+def test_fit_transform_values_out_of_borders(
+    ts_check_pipeline_with_limit_transform: TSDataset, lower_bound: float, upper_bound: float, n_test: str
+):
+    """Check that Exception raises when there are values out of bounds"""
+    transform = LimitTransform(in_column="target", lower_bound=lower_bound, upper_bound=upper_bound)
+    if n_test == "2":
```

Comment:
There should't be any additional logic in the test, you can handle it via parametrisation